server/routes/bookAVisit.ts

```diff
@@ -123,9 +125,50 @@ export default function routes(
         return closedVisit || visitor.restrictions.some(restriction => restriction.restrictionType === 'CLOSED')
       }, false)
       visitSessionData.visitRestriction = closedVisitVisitors ? 'CLOSED' : 'OPEN'
-
       visitSessionData.closedVisitReason = closedVisitVisitors ? 'visitor' : undefined
 
+      const closedVisitPrisoner = visitSessionData.prisoner.restrictions.reduce((closedVisit, restriction) => {
```

Comment:
Might be clearer to use a `.some` rather than `.reduce`? Would make it clearer what it's doing?

Reply:
Good spot! Have updated to make it simpler.